test/accessibility/conftest.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from pytest_axe.pytest_axe import PytestAxe as Axe
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+base_url = "https://screenshots.stage.mozaws.net/"
+
+_panel_button_locator = "pageActionButton"
+_take_screenshot_button_locator = (By.ID, "pageAction-panel-screenshots_mozilla_org")
+
+
+def wait_for_element_to_load(driver, selector):
+    wait = WebDriverWait(driver, 10)
+    element = wait.until(EC.presence_of_element_located(selector))
+    return element
+
+
+@pytest.fixture(scope="class")
+def home_page():
+    """Launch Screenshots home page as guest."""
+    profile = webdriver.FirefoxProfile()
```

Comment:
I'm pretty sure this is testing the version of Screenshots that packaged with Firefox, but won't test the development version. I.e., if you fix an error locally, this test won't see your fix.

We have code in our tests that handles this: https://github.com/mozilla-services/screenshots/blob/master/test/test.js#L54-L85 – if this was written in Node.js you could use that, though I suppose it could also be translated to Python.

Reply:
Noted. I'll look at the code you shared.